server/services/expectedArrivalsService.ts

```diff
@@ -28,13 +36,27 @@ export default class ExpectedArrivalsService {
     return expectedArrivals.sort(compareByFullName)
   }
 
+  private async getRecentArrivals(agencyId: string): Promise<RecentArrival[]> {
+    const token = await this.hmppsAuthClient.getSystemClientToken()
+    const welcomeClient = this.welcomeClientFactory(token)
+    const recentArrivals = await welcomeClient.getRecentArrivals(agencyId)
+    return recentArrivals.sort(compareByDateAndTime)
+  }
+
   private async getTransfers(agencyId: string): Promise<Arrival[]> {
     const token = await this.hmppsAuthClient.getSystemClientToken()
     const welcomeClient = this.welcomeClientFactory(token)
     const transfers = await welcomeClient.getTransfers(agencyId)
     return transfers.map(transfer => ({ ...transfer, fromLocationType: LocationType.PRISON })).sort(compareByFullName)
   }
 
+  public async getRecentArrivalsGroupedByDate(agencyId: string): Promise<Map<string, RecentArrival[]>> {
+    const recentArrivals = await this.getRecentArrivals(agencyId)
+    return groupBy(recentArrivals, (recentArrival: RecentArrival) =>
+      moment(recentArrival.movementDateTime).format('dddd D MMMM')
+    )
+  }
```

Comment:
sweet -  if we were to push stuff down here, we'd have to do something funky creating a range of dates